### ISSUE TITLE
Feature Request: Inbuilt support for C/C++ & Java

### ISSUE THREAD

**Issue by 231tr0n** [chrisgrieser/nvim-chainsaw#15]
### Feature Requested

Hi chrisgrieser! Thank you for developing and maintaining this wonderful and compact plugin. Is it possible to add inbuilt support for the following languages as well to this plugin.
- C/C++
- Java
- Go 

### Checklist

- [X] The feature would be useful to more users than just me.

**Comment by chrisgrieser** [comment]
> Is it possible to add inbuilt support for the following languages as well to this plugin.

Yes, it is, but since I have no experience in those languages, some else will have to make a PR for that.

It's relatively straightforward, adding the correct template to the table in this file: https://github.com/chrisgrieser/nvim-chainsaw/blob/main/lua/chainsaw/log-statements-data.lua

(Closing since not actionable for me, but as mentioned, PRs are welcome.)

**Comment by andrewferrier** [comment]
FYI - `debugprint` has support for these. You are welcome to adapt the configuration from [here](https://github.com/andrewferrier/debugprint.nvim/blob/684846f982d2fb2833c5815f5de3152a7da9e98b/lua/debugprint/filetypes.lua#L96) into `nvim-chainsaw` if it's helpful.

**Comment by chrisgrieser** [comment]
@andrewferrier Thanks for the offer! However, I am quite hesitant to write log statements for languages I am not familiar with myself, since it's easy to make mistakes then. 

And since the config from debugprint is structured very differently from how chainsaw structures them, I am not even sure if I could properly translate a c++ log statement correctly from debugprint's to chainsaw's format.

I think it would really be better if some who actually knows those languages made a PR to chainsaw 😅

**Comment by andrewferrier** [comment]
Fair enough, thought I'd offer :) I know `debugprint` is MIT licensed so it's not like technically you need my permission, but best to be open anyway. I'll leave this with you then...

**Comment by itse4elhaam** [comment]
bump!

Would love to see this happening!
